src/pages/Checkout/CheckoutPage.vue

```diff
@@ -428,15 +432,29 @@ export default {
 				this.setUpdatingTotals(false);
 			});
 		},
+		completeTransaction(transactionId) {
+			// compile transaction information
+			const transactionData = {
+				transactionId,
+				itemTotal: this.totals.itemTotal,
+				loans: _map(this.loans, loan => {
+					return _pick(loan, ['__typename', 'id', 'price']);
+				}),
+				donations: _map(this.donations, donation => {
+					return _pick(donation, ['__typename', 'id', 'price']);
+				}),
+			};
+			// fire transaction events
+			this.$kvTrackTransaction(transactionData);
+			// redirect to thanks
+			window.setTimeout(
+				this.redirectToThanks(transactionId),
```

Comment:
IIUC this calls redirectToThanks and passes its result to setTimeout, rather than enqueueing a call. Is that right?

Reply:
Good call @remoun Thanks!